manifest.json

```diff
@@ -1,6 +1,21 @@
 {
-    "name": "Hello, world!",
+    "name": "Totally for real actual ghostery extension",
     "version":"0.000001",
-    "manifest_version": 2
+    "manifest_version": 2,
+    "content_scripts": [
+        {
+            "matches": ["<all_urls>"],
+            "js": ["content.js"]
```

Comment:
It's good to give your content scripts descriptive names and use a `content` folder instead of the filename to make their function clear. Here you might call it something like `content-scripts/helpful-notification.js`. Take a look at how we organize them in the extension.

Reply:
'content.js' renamed to 'friendly-notification.js', moved into a folder titled 'content-scripts'